src/ert/gui/simulation/run_dialog.py

```diff
@@ -403,15 +408,15 @@ def _on_event(self, event: object) -> None:
             self._show_done_button()
         elif isinstance(event, FullSnapshotEvent):
             if event.snapshot is not None:
-                self._snapshot_model._add_snapshot(event.snapshot, event.iteration)
+                self._snapshot_model._add_snapshot(event.snapshot, str(event.iteration))
```

Comment:
Is it possible to change the event type to have iteration as a string instead of int, so we dont have to parse it every time? Would it break something somewhere else?

Reply:
I've proposed to revert the last two commits, and rather have these as a separate PR.
We are snowballing atm, and this type change str vs int makes sense to resolve, just not in this one. Personal opinion.

Reply:
Agreed

Reply:
They have now been removed, just mentioning this so no misunderstanding.